pkg/component/status/status.go

```diff
@@ -83,7 +86,7 @@ func (s *Status) Init(_ context.Context) error {
 	}
 
 	removeLeftovers(s.Socket)
-	s.listener, err = net.Listen("unix", s.Socket)
+	s.listener, err = listen("unix", s.Socket)
```

Comment:
Maybe we can encapsulate the "unix" part also into the listen function, so that it only accepts the path to the socket/pipe.